content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Processing
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is is currently in private beta. If you would like to apply to it, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+
+To access the processing panel please use the upper left menu:
```

Comment:
> To access the processing panel please use the upper left menu:

To access the processing panel, use the upper left menu:

[omit "please"]

Reply:
👍